lua/lspconfig/util.lua

```diff
@@ -20,9 +20,6 @@ M.default_config = {
 M.on_setup = nil
 
 function M.bufname_valid(bufname)
-  if not bufname then
-    return false
-  end
   if bufname:match '^/' or bufname:match '^[a-zA-Z]:' or bufname:match '^zipfile://' or bufname:match '^tarfile:' then
```

Comment:
if `bufname` is nil this will error out, is that intentional?

Reply:
look at the source code, I don't think it can be nil.

https://github.com/neovim/neovim/blob/master/src/nvim/api/buffer.c#L1037-L1048